--- a/include/package-pack.mk
+++ b/include/package-pack.mk
@@ -583,1 +583,1 @@ $(_define) Package/$(1)/CONTROL
-	  --info "depends:$$(foreach depends,$$(subst $$(comma),$$(space),$$(subst $$(space),,$$(subst $$(paren_right),,$$(subst $$(paren_left),,$$(Package/$(1)/DEPENDS))))),$$(depends))" \
+	  --info "depends:$$(foreach conflict,$$(call mergelist,$$(CONFLICTS)),!$$(conflict)) $$(foreach depends,$$(subst $$(comma),$$(space),$$(subst $$(space),,$$(subst $$(paren_right),,$$(subst $$(paren_left),,$$(Package/$(1)/DEPENDS))))),$$(depends))" \
